### ISSUE TITLE
Alternative Admin Role not mapping as expected

### ISSUE THREAD

**Issue by Shinerrs** [opensearch-project/security#535]
create a custom admin role just for the sake of the name.

I want my role to give admin access including access too OpenDistro tools.

My understanding is that I have a RoleMapping  called "admin" that gives role access to "all_access" role.

When I do the following, neither seem to be happening:

>     logging_access_admin:
>       reserved: false
>       backend_roles:
>       - "admin"
>       - "logging_access_admin"
>       - "all_access"

Anyone know what I am doing wrong. 

My only idea is, that there are other options to use instead of backend_role, and that could link by roleMapping to the correct permissions. But so far, the documentation lacks to show the dynamic of this config. If anyone can help, I would be grateful.

Basically what I want is, logging_access_admin to have the same permissions as the hidden roleMapping called "admin".

**Comment by vrozov** [comment]
From the issue description it is not clear what is not working and what you are trying to accomplish.

**Comment by Anthony7774** [comment]
@Shinerrs The logging_access_admin would need to have the same permissions as all_access role. To which you can then map users with backend role "admin". Be sure to update opendistro_security.restapi.roles_enabled: ["logging_access_admin"]
(or plugins.security.restapi.roles_enabled if you upgraded to opensearch)

**Comment by davidlago** [comment]
We are doing some "spring cleaning in the fall", and to make sure we focus our energies on the right issues and we get a better picture of the state of the repo, we are closing all issues that we are carrying over from the ODFE era (ODFE is no longer supported/maintained, see post [here](https://opendistro.github.io/for-elasticsearch/blog/2022/08/Open-Distro-is-archived/)). 

If you believe this issue should still be considered for current versions of OpenSearch, apologies! Please let us know by re-opening it.

Thanks!
